### ISSUE TITLE
How can I uninstall this plugin?

### ISSUE THREAD

**Issue by mrmlnc** [adrenth/redirect#22]
Hello, @adrenth,

Due to circumstances, I no longer need this plugin and I would uninstall it. But when I click on the delete button - I get an error.

OctoberCMS version: 350
Redirect version: 2.0.6

```php
exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to protected Adrenth\Redirect\Classes\PublishManager::__construct() from context 'Adrenth\Redirect\Updates\Publish'' in /public_html/plugins/adrenth/redirect/updates/publish.php:23
```

**Comment by adrenth** [comment]
Thanks for reporting this issue. Should be a small fix. In the next release 2.0.7 it will be fixed. I've planned this release tommorrow.

**Comment by adrenth** [comment]
Hi Denis,

I just released 2.0.7 which should fix this issue.

If you still have issues uninstalling the plugin 

Remove this directory `/plugins/adrenth/redirect`
Remove this file `/storage/app/redirects.csv` (default location of storage)

And execute these queries:
````
SET FOREIGN_KEY_CHECKS = 0;
DROP TABLE `adrenth_redirect_clients`;
DROP TABLE `adrenth_redirect_redirects`;
SET FOREIGN_KEY_CHECKS = 1;

DELETE FROM `system_plugin_versions` WHERE `code` = 'Adrenth.Redirect';
DELETE FROM `system_plugin_history` WHERE `code` = 'Adrenth.Redirect';
````

Thanks for using my plugin, I hope you have some time to leave a review on the OctoberCMS website (http://octobercms.com/plugin/adrenth-redirect).

**Comment by mrmlnc** [comment]
Great work. It works like a dream. @adrenth, thank you.
